pkg/cmd/admin/node/node_options.go

```diff
@@ -179,11 +179,31 @@ func (n *NodeOptions) GetPrinters(gvk unversioned.GroupVersionKind) (kubectl.Res
 		return nil, nil, err
 	}
 
-	printerWithHeaders, err := n.Printer(mapping, false, false, false, false, false, false, []string{})
+	printerWithHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{
+		NoHeaders:          false,
+		WithNamespace:      false,
+		WithKind:           false,
+		KindName:           "",
+		Wide:               false,
+		ShowAll:            false,
+		ShowLabels:         false,
+		AbsoluteTimestamps: false,
+		ColumnLabels:       []string{},
+	})
 	if err != nil {
 		return nil, nil, err
 	}
-	printerNoHeaders, err := n.Printer(mapping, true, false, false, false, false, false, []string{})
+	printerNoHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{
```

Comment:
Here it would be:

```go
printerWithHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{NoHeaders: true})
```

Refactoring the signature of that method eventually is simplifying some code that needed to pass many arguments, now just need to set the options that we need to differ from default :)